superset-frontend/cypress-base/cypress/integration/explore/AdhocMetrics.test.ts

```diff
@@ -70,12 +72,16 @@ describe('AdhocMetrics', () => {
     cy.get('[data-test=metrics]').find('.option-label').last().click();
 
     // add custom SQL
-    cy.get('#adhoc-metric-edit-tabs-tab-SQL').click();
-    cy.get('#metrics-edit-popover').find('.ace_content').click();
-    cy.get('#metrics-edit-popover')
+    cy.get('[data-test="adhoc-metric-edit-tabs"]')
+      .contains('Custom SQL')
+      .click();
+    cy.get('[data-test="metrics-edit-popover"]')
       .find('.ace_text-input')
       .type('/COUNT(DISTINCT name)', { force: true });
-    cy.get('#metrics-edit-popover').find('button').contains('Save').click();
+    cy.get('[data-test="metrics-edit-popover"]')
+      .find('[data-test="AdhocMetricEdit#save"]')
+      .contains('Save')
```

Comment:
do we still need the lookups by text if we're using the data-test attributes?

Reply:
Also wondering if we need the `get`->`find` chain here, or if we can just `get` the specific element we want to test on, with a specific enough attribute